packages/nextjs/src/middlewareClient.ts

```diff
@@ -82,6 +90,10 @@ export function createMiddlewareClient<
 		cookieOptions?: CookieOptionsWithName;
 	} = {}
 ): SupabaseClient<Database, SchemaName, Schema> {
+	console.warn(
```

Comment:
maybe `console.debug` instead? not sure if we want to repeat the mistake of flooding the console with a warning here

Reply:
Removed it.